assets/section-rich-text.css

```diff
@@ -1,38 +1,37 @@
 .rich-text {
-  margin-left: auto;
-  margin-right: auto;
-  text-align: center;
   z-index: 1;
 }
 
-.rich-text.rich-text--full-width {
-  max-width: initial;
-  width: 100%;
+.rich-text__wrapper {
+  display: flex;
+  justify-content: center;
+  width: calc(100% - 4rem / var(--font-body-scale));
 }
 
-.rich-text__blocks {
+.rich-text:not(.rich-text--full-width) .rich-text__wrapper {
   margin: auto;
-  /* 2.5rem margin on left & right */
-  width: calc(100% - 5rem / var(--font-body-scale));
-}
-
-.rich-text__blocks * {
-  overflow-wrap: break-word;
+  width: calc(100% - 8rem / var(--font-body-scale));
 }
 
-.rich-text--full-width .rich-text__blocks {
-  /* 4rem (1.5rem + 2.5rem) margin on left & right */
-  width: calc(100% - 8rem / var(--font-body-scale));
+.rich-text__blocks {
+  width: 100%;
 }
 
 @media screen and (min-width: 750px) {
```

Comment:
Not related to your PR but because I noticed it here, I'll leave it as a reminder for us to review this later and create an issue for this.

The setting is called `Desktop content position` and applies to "tablet and above" (> `750px`). It looks we're consistently doing that in many other sections like Image with Text, Image Banner and others. Sounds like this is the intended behaviour.

But I just noticed that on the Product page, the `Desktop layout` applies to > `750px` and the `Desktop media size` applies to > `990px`. It uses the same `automatically optimized for mobile` description that is mentioned in the setting you just added, so I'm wondering if there are more occurrences of this inconsistency across the theme.

<img width="276" alt="image" src="https://user-images.githubusercontent.com/37168033/167217776-e371db4e-366d-46c9-9be7-06c6ebbd8a59.png">